models/nft/ethereum/nft_ethereum_schema.yml

```diff
@@ -168,3 +168,32 @@ models:
         description: "Name of protocol that routed this trade using the aggregator, is NULL if it's the same as aggregator_name"
       - name: hash_marker_size
         description: "Number of characters the hash_marker is made of, can be useful on joins"
+  
+  - name: nft_ethereum_top_holders
+    meta:
+      blockchain: ethereum
+      sector: nft
+      contributors: Henrystats
+    config:
+      tags: ['nft', 'ethereum', 'top Holder']
+    description: >
+        Top 50 NFT Holders
+    tests:
+      - dbt_utils.unique_combination_of_columns:
+          combination_of_columns:
+            - nft_contract_address
```

Comment:
this is the test that is failing, although I cannot imagine why it fails. My first impression was that you're calculating `rn` separately for each token standard and then unioning, so it could be causing duplicates... but I can't imagine how the same nft_contract_address would exist in both token standards?

I'm pretty puzzled by why this is failing.

Reply:
yeah it's odd. I'll be taking a look as I plan to rewrite this as an incremental version. Although the balances table is actually returning inaccurate data and the query looks correct.

I mean the balances_erc72_latest that's on dune already. I'm trying to see if I can look for a small address with minimal transactions so I can see where the error is.

the total supply is accurate but the balances isn't.